<launch>
  <arg name="sim" default="true"/>

  <include file="$(find asl_turtlebot)/launch/root.launch">
    <arg name="world" value="project_42" />
    <arg name="x_pos" default="3.15"/>
    <arg name="y_pos" default="1.6"/>
    <arg name="z_pos" default="0.0"/>
    <arg name="rviz" default="section4"/>
    <arg name="model" default="asl_turtlebot"/>
    <arg name="sim" default="$(arg sim)"/>
    <arg name="detector" default="detector_mobilenet.py" />
  </include>

  <node pkg="asl_turtlebot" type="navigator.py" name="navigator" output="screen" />
  <!-- <node pkg="asl_turtlebot" type="pose_controller.py" name="turtlebot_pose_controller" /> -->
  <!-- <node pkg="asl_turtlebot" type="detector.py" name="detector" /> -->
  <!-- <node pkg="asl_turtlebot" type="detector_mobilenet.py" name="detector_mobilenet" /> -->
  <node pkg="asl_turtlebot" type="pet_logger.py" name="pet_logger" output="screen"/>
</launch>
